### DISCUSSION TITLE
Hey!How can I search???

### DISCUSSION THREAD

**Discussion by allMagicNB** [community/community#118694] [Mobile]
After trying to search for it a lot, I found that searching with this thing yielded something completely unrelated to my search term......

**Comment by allMagicNB** [comment]
For example:
![Screenshot_20240411_215457_com.github.android.jpg](https://github.com/community/community/assets/116283787/78c99a76-16f3-4975-bbd1-bf54085fdc3c)
I searched for the title of this issue, but I don't see this issue in the results
![Screenshot_20240411_214121_com.github.android.jpg](https://github.com/community/community/assets/116283787/4a29c8a1-d82c-4956-958e-4bc86c4a4ef0)

**Comment by ciozru** [comment]
I could help you if I knew Chinese. Explain the problem in more detail.

**Reply by allMagicNB** [reply]
eh…
Like search "hello",but search results without "hello".
but…There is an issue titled "hello"???
